### PR TITLE
Fix controller_manager update rate

### LINKED ISSUE #343: TurtleBot3 simulation doesn't work at 50Hz as specified
Given:
- https://github.com/cyberbotics/webots_ros2/blob/c1b3a055f9b3d931066083e561c3dbee5b386c3a/webots_ros2_turtlebot/resource/ros2control.yml#L3
- https://github.com/cyberbotics/webots_ros2/blob/c1b3a055f9b3d931066083e561c3dbee5b386c3a/webots_ros2_turtlebot/resource/ros2control.yml#L19

We should receive Odometry messages at 50Hz, but instead, we get them at 30Hz.

This is reasonable as a simulation with a `basicTimeStep` at 32ms cannot deliver that high frequency. However, reducing the `basicTimeStep` increases the frequency to 60Hz which is wrong.

This indicates there is a bug in `webots_ros2_control`.

### PR DESCRIPTION
**Pull request by lukicdarkoo**
Fixes #343

There is not much we can do to tackle this. Basically:
- If basicTimeStep doesn't match the update rate then we notify the user about that.
- If a user uses low basicTimeStep then we skip some updates.

**Tasks**
- [x] Fix the update_rate
- [x] Fix copyright 
- [x] Fix CI

### CODE REVIEW

**Comment by lukicdarkoo** [comment]
- Rolling CI fail: Rolling never executes the `generate_test_description()` function. It seems something is updated, but I cannot find what. Investigating...
- Galactic (main): This should be temporary as Galactic (testing) works fine.

**Comment by lukicdarkoo** [comment]
We got affected by https://github.com/ros2/launch_ros/pull/278. I believe we didn't use the tests in the correct way.

**Comment by lukicdarkoo** [comment]
- `CI / industrial_ci (main, galactic) (pull_request)`: It seems they pushed a wrong header (looks fixed in `testing`)
- `CI / industrial_ci (testing, foxy) (pull_request)`: Standard random error caused by RViz and the Turtlebot3 package.

File: webots_ros2_turtlebot/worlds/turtlebot3_burger_example.wbt [hunk 1]
```diff
@@ -4,6 +4,7 @@ WorldInfo {
     "Simple TurtleBot3 Burger simulation that can be controlled by a ROS."
   ]
   title "ROS2 simulation of the TurtleBot3 Burger robot"
+  basicTimeStep 20
```
Comment (BenjaminHug8) [inline webots_ros2_turtlebot/worlds/turtlebot3_burger_example.wbt:7]: Same as `Tiago`.

File: webots_ros2_turtlebot/resource/ros2control.yml [hunk 2]
```diff
@@ -16,5 +16,4 @@ diffdrive_controller:
     wheel_separation: 0.160
     wheel_radius: 0.033
 
-    publish_rate: 50.0
```
Comment (BenjaminHug8) [inline webots_ros2_turtlebot/resource/ros2control.yml:19]: Same as other `ros2_control.yml`.

File: webots_ros2_tiago/worlds/default.wbt [hunk 3]
```diff
@@ -3,7 +3,7 @@ WorldInfo {
   info [
     "An office break room, surrounded by office desks."
   ]
-  basicTimeStep 16
+  basicTimeStep 20
```
Comment (BenjaminHug8) [inline webots_ros2_tiago/worlds/default.wbt:6]: Same as `rats_life`.

File: webots_ros2_epuck/worlds/rats_life_benchmark.wbt [hunk 4]
```diff
@@ -3,7 +3,7 @@ WorldInfo {
   info [
     "Rat's life pattern"
   ]
-  basicTimeStep 16
+  basicTimeStep 20
```
Comment (BenjaminHug8) [inline webots_ros2_epuck/worlds/rats_life_benchmark.wbt:6]: Same as `e-puck`.

File: webots_ros2_epuck/resource/ros2_control.yml [hunk 5]
```diff
@@ -16,5 +16,4 @@ diffdrive_controller:
     wheel_separation: 0.05685
     wheel_radius: 0.02
 
-    publish_rate: 50.0
```
Comment (BenjaminHug8) [inline webots_ros2_epuck/resource/ros2_control.yml:19]: Do we simply remove it as it has no effect and I will put it again when my PR on `ros2control` is accepted, or should we keep it anyway?

File: webots_ros2_tiago/resource/ros2_control.yml [hunk 6]
```diff
@@ -16,7 +16,6 @@ diffdrive_controller:
     wheel_separation: 0.5
     wheel_radius: 0.12
 
-    publish_rate: 50.0
```
Comment (BenjaminHug8) [inline webots_ros2_tiago/resource/ros2_control.yml:19]: Same as other `ros2_control.yml`.

File: webots_ros2_epuck/worlds/epuck_world.wbt [hunk 7]
```diff
@@ -4,6 +4,7 @@ WorldInfo {
     "Simple e-puck simulation that can be controlled with ROS2."
   ]
   title "ROS2 simulation of the e-puck robot"
+  basicTimeStep 20
```
Comment (BenjaminHug8) [inline webots_ros2_epuck/worlds/epuck_world.wbt:7]: Do we have to keep this `basicTimeStep`? I think it was set only for testing?

**Review by BenjaminHug8 — CHANGES_REQUESTED** [review]
Seems all good, I just have 1 question and 1 suggestion that occur multiple times.

File: webots_ros2_epuck/resource/ros2_control.yml (hunk 5 above)
Comment (lukicdarkoo) [reply webots_ros2_epuck/resource/ros2_control.yml:19]: I would rather put it later.

File: webots_ros2_epuck/worlds/epuck_world.wbt (hunk 7 above)
Comment (lukicdarkoo) [reply webots_ros2_epuck/worlds/epuck_world.wbt:7]: If we want to have 50Hz publish rate then we have to put it.

File: webots_ros2_epuck/resource/ros2_control.yml (hunk 5 above)
Comment (BenjaminHug8) [reply webots_ros2_epuck/resource/ros2_control.yml:19]: Fine for me then.

File: webots_ros2_epuck/worlds/epuck_world.wbt (hunk 7 above)
Comment (BenjaminHug8) [reply webots_ros2_epuck/worlds/epuck_world.wbt:7]: Ok fine for me too.

**Review by BenjaminHug8 — APPROVED** [review]
Thank you.

**Pull request merged**